test/soca/test_bmat.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+project_binary_dir=$1
+project_source_dir=$2
+
+source ${project_source_dir}/test/soca/test_utils.sh
+
+# Clean test files created during a previous test
+echo "============================= Test that the grid has been generated"
+rm -f  ${DATA}/soca_gridspec.nc
+rm -f  ${DATA}/ocn.bkgerr_stddev.incr.2018-04-15T09:00:00Z.nc
+rm -f  ${DATA}/Data/ocn.iter1.incr.2018-04-15T09:00:00Z.nc
```

Comment:
I'm not sure L13-L18 should be in this script. You're removing files that are created by the next test.